prov/efa/src/rdm/rxr_ep.c

```diff
@@ -168,6 +168,7 @@ struct rxr_op_entry *rxr_ep_alloc_rx_entry(struct rxr_ep *ep, fi_addr_t addr, ui
 	rx_entry->efa_outstanding_tx_ops = 0;
 	rx_entry->shm_outstanding_tx_ops = 0;
 	rx_entry->op = op;
+	rx_entry->peer_rx_entry.srx = peer_srx;
```

Comment:
Also just do `rx_entry->peer_rx_entry.srx = ep->srx` here as suggested, and remove the additional input

Reply:
fixed